modules/virt-creating-vm-from-snapshot-web.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/creating_vms_rh/creating-vms-from-instance-types.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-vm-from-snapshot-web_{context}"]
+= Creating a VM from an existing snapshot by using the web console
+
+You can create a new VM by copying an existing snapshot.
+
+.Procedure
+
+. Navigate to *Virtualization* -> *VirtualMachines* in the web console.
```

Comment:
```suggestion
. Go to *Virtualization* -> *VirtualMachines* in the web console.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n

Reply:
All of our CNV web console procedures use "Navigate," so I will leave this step as is.